### PR TITLE
refactor: structure, styles. scss features added

### PR DESCRIPTION
**Pull request by Aleksey-Bortnik**
have some new bugs, but it in the progress...

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
-.env
\ No newline at end of file
+.env
+dist
```
Comment (ElizavetaSelivanova) [inline .gitignore]: add empty line at the end of file

File: dist/style/main.css [hunk 2]
```diff
@@ -1,6 +1,11 @@
+.header__section-container {
```
Comment (ElizavetaSelivanova) [inline dist/style/main.css]: REMOVE dist/ folder from the repository, add it to .gitignore

File: index.html [hunk 3]
```diff
@@ -134,7 +146,7 @@ <h6 class="designers__section-description">Custom Design designers</h6>
       <div class="product-manager__section-description">
         <p class="product-manager-title-shift">Easier decision making for</p>
 
-        <h3 class="product-manager__section-title, product-manager-title-shift">Product Managers</h3>
+        <h2 class="product-manager__section-title, product-manager-title-shift">Product Managers</h2>
```
Comment (ElizavetaSelivanova) [inline index.html]: Remove ',' between classes

File: index.html [hunk 4]
```diff
@@ -184,46 +199,52 @@ <h6 class="marketers__section-title">Custom Design designers</h6>
...
         </article>
 
         <article>
           <div class="strategies__section-image-time"></div>
           <p>By <span class="strategies__section-accent">Abdullah |</span> 03 March 2019</p>
-          <h3 class="strategies__section-title">Success Steps For Your Personal Or Business Life</h3>
+          <h4 class="strategies__section-title">Success Steps For Your Personal Or Business Life</h4>
         </article>
       </div>
       <div class=" strategies__section-carousell">
```
Comment (ElizavetaSelivanova) [inline index.html]: You can use 
`<ul> <li>...</li> ... </ul>` for dots

File: index.html [hunk 5]
```diff
@@ -239,7 +260,7 @@ <h4 class="footer__description">Legal Stuff</h4>
           <a class="footer__links-decorator" href="#">Privacy Policy</a>
           <a class="footer__links-decorator" href="#">Terms of Service</a>
         </div>
-        <div class="footer__input-container">
+        <div id="registration" class="footer__input-container">
```
Comment (ElizavetaSelivanova) [inline index.html]: Why did you add 'id' here?

File: index.html [hunk 6]
```diff
@@ -184,46 +199,52 @@ <h6 class="marketers__section-title">Custom Design designers</h6>
...
         <div class="circle">
         </div>
         <div class="circle">
         </div>
         <div class="circle">
         </div>
     </section>
+  </div>
 
+  <div class="footer__section-container">
```
Comment (ElizavetaSelivanova) [inline index.html]: <footer> tag should be before this tag, if needed, you can add needed class there

File: source/scss/abstracts/_mixins.scss [hunk 7]
```diff
@@ -0,0 +1,21 @@
+@mixin text-formatter($font-size: 16px, $line-height: 28px) {
+  font: {
+    family: Poppins;
```
Comment (ElizavetaSelivanova) [inline source/scss/abstracts/_mixins.scss:3]: Create variables for font-family

File: source/scss/abstracts/_mixins.scss [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+    style: normal;
+    weight: 500;
+    size: $font-size;
+  }
+  line-height: $line-height;
+  color: red; //testing color
```
Comment (ElizavetaSelivanova) [inline source/scss/abstracts/_mixins.scss]: Use variables for colors

File: source/scss/components/_banner.scss [hunk 9]
```diff
@@ -3,28 +3,23 @@
   width: 100%;
   height: 100%;
 }
+
 .banner__section-title {
```
Comment (ElizavetaSelivanova) [inline source/scss/components/_banner.scss]: Use scss useful features 
```
.banner {
  &__section-title {}
// or
  &__section {
    &-image {}
    &-title {} 
  }
}
```

File: source/scss/components/_banner.scss [hunk 10]
```diff
@@ -33,38 +28,31 @@
 .banner__section-buttons-container {
   display: flex;
   margin-top: 45px;
+  @include text-formatter;
 }
 
 .banner__section-start-button {
   width: 179px;
```
Comment (ElizavetaSelivanova) [inline source/scss/components/_banner.scss]: Try to not use concretize sizes for buttons, use paddings

File: source/scss/components/_design.scss [hunk 11]
```diff
@@ -1,39 +1,24 @@
...
+    @include text-formatter;
   }
 }
 
 .designers__section-image {
-  background-image: url("../../source/img/design/design.png");
+  background-image: url("../../source/assets/img/design/design.png");
   width: 520px;
   height: 573px;
   margin-left: 5%;
```
Comment (ElizavetaSelivanova) [inline source/scss/components/_design.scss]: Why margin-left 5% here?

File: source/scss/components/_header.scss [hunk 12]
```diff
@@ -1,6 +1,12 @@
+.header__section-container {
+  display: flex;
+  // justify-content: center;
```
Comment (ElizavetaSelivanova) [inline source/scss/components/_header.scss]: Remove unnecessary comments

File: source/scss/main.scss [hunk 13]
```diff
@@ -14,14 +18,55 @@
...
+  justify-content: center;
+  flex-direction: column;
+ width: 100%;
 }
 
-.site__container {
-  margin: 0 auto;
-  max-width: 1140px;
-  font-family: Poppins;
+.header__section-container,
```
Comment (ElizavetaSelivanova) [inline source/scss/main.scss]: Add common class for these sections to avoid classes listing

**Review by ElizavetaSelivanova — CHANGES_REQUESTED** [review]

**Comment by ElizavetaSelivanova** [comment]
Good job!

One more comment - resolve conflicts, please.

File: source/scss/components/_design.scss [hunk 14]
```diff
@@ -1,39 +1,24 @@
...
 
 .designers__section-description {
   max-width: 486px;
-  font-family: Poppins;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 16px;
-  line-height: 26px;
-  color: #2f281e;
+  @include text-formatter;
```
Comment (irina-chernik) [inline source/scss/components/_design.scss]: with @include you still add the code to all places mixin is used
all font properties in CSS are inherited so that's why base font-family and font-size are usually set on body selector
then font properties are specified for other element if they have different styling
no need to add the same styles everywhere

File: index.html [hunk 15]
```diff
@@ -153,17 +165,20 @@ <h3 class="product-manager__section-title, product-manager-title-shift">Product
         </div>
 
         <div class="product-manager__section-text-wrapper">
-          <div class="product-manager__section-icon"></div>
+          <div class="product-manager__section-icon">
```
Comment (irina-chernik) [inline index.html]: use &lt;i&gt; tag for icons

File: index.html [hunk 16]
```diff
@@ -153,17 +165,20 @@ <h3 class="product-manager__section-title, product-manager-title-shift">Product
...
           <p class="product-manager__section-text">We track the markets daily and know where the savings are.</p>
         </div>
       </div>
     </section>
+  </div>
 
+  <div class="marketers__section-container">
     <section class="marketers__section">
       <div class="marketers__section-description">
         <p class="marketers__section-text">Optimization for</p>
```
Comment (irina-chernik) [inline index.html]:
```
<h2>
<span class="marketers__section-text">Optimization for</span>
Marketers
</h2>
```
all the text is header by meaning and part of it styled differently
